chat-microservice/app/api/v1/chat.py

```diff
@@ -61,3 +62,31 @@ async def get_user_chats(current_user_id = Depends(get_current_user_id)):
     """
     results = Chat.objects().filter(users_id__contains=str(current_user_id)).allow_filtering()
     return list(results)
+
+@router.get(
+    "/chat/messages/", 
```

Comment:
Route should be /chat/:id/messages/ and get the chat id from the url path and not from query param